src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -7,7 +7,7 @@ export default (accessToken, userID, opts) => {
   const gravityLoader = gravityLoaderWithAuthenticationFactory(gravityAccessTokenLoader)
 
   return {
-    authenticatedArtworkLoader: gravityLoader(id => `artwork/${id}`),
+    artworkLoader: gravityLoader(id => `artwork/${id}`),
```

Comment:
As discussed w/ @sweir27 in Slack: https://artsy.slack.com/archives/CA8SANW3W/p1553110651795100 , we want to serve fresh (uncached) results for artworks in order for an accurate 'Sold' message to be promptly displayed.

So by calling this `artworkLoader` we will get that uncached behavior for artworks for logged in users (which we're ok with).

Other cases, like making sure there's up to date bid/order information either comes from authenticated loaders, or the [sale artwork loader](https://github.com/artsy/metaphysics/blob/a6896dbd361cd45a713f086119cd4dc30998f0c1/src/lib/loaders/loaders_without_authentication/gravity.ts#L65) which is uncached as well.

Reply:
If I understand correctly, this will cause authenticated queries that include artwork properties to fall through to the API instead of serving data from the cache. Isn't that a potentially large number of requests that would now be drastically slower? I might prefer just opting for a shorter cache expiration in this case.

If I didn't understand, just ignore me.

Reply:
Any request that fetches the artwork individually, yes.

This may drastically increase the number of requests to `/api/v1/artwork/:id` , but I think that might be fine (assuming this is a requirement for getting all these cache changes in). That's a single entity response from Gravity, where it's cached via garner already.

We can also just experiment with shortening the cache TTL for this loader, from 60 seconds to...something else?

Reply:
Maybe we can measure the performance impact of this?

Another option could be to pass an option in these queries so we can choose.

Or, would be extra cool if we could automatically return a fresh result only if you are authenticated + requesting any of the "timely" fields like `is_offerable` or `for_sale`, but otherwise use the cached version. 😬

Reply:
So effectively, if I understand this correctly, we're not caching artworks for any authenticated queries?

Reply:
Correct, and the reason was b/c we want it to say 'Sold' right away (and not the 60 seconds it would take w/ this new caching).

@sweir27 I think we're getting too speculative with the changes we want...is it possible to _just_ roll this out on staging and QA and get feedback on what _actually_ is stale and what we should actually change? Otherwise I don't have confidence we're not overcomplicating things and reducing optimizations when we're still not sure we need to?

Reply:
@mzikherman I just spoke with @joanna20 and she agrees that the ideal behavior would be to, for logged-in users, always show the most up-to-date purchasing options. (so, what we've discussed).

I suggest that we ship your current change (maybe to a review app?) and analyze the performance implications in staging if that's what @joeyAghion's worried about, since I don't think this is a premature optimization.